### PR TITLE
Anticipate the lbp of comdex

### PR DESCRIPTION
**Pull request by Thunnini**
I expect that the pool id of comdex may be 600.
Please check that the chain info and channel info is correct and the pool id is correct.

### CODE REVIEW

File: src/stores/root.ts [hunk 1]
```diff
@@ -229,6 +229,13 @@ export class RootStore {
 					spotPriceDestDenom: 'uosmo',
 					destCoinId: 'pool:uosmo',
 				},
+				{
```
Comment (Thunnini) [inline src/stores/root.ts:232]: The price of base coin (comdex) will be shown on the LBP section. And, it should be important to user who is considering buying comdex. So, please check that this is correct.

File: src/config.ts [hunk 2]
```diff
@@ -48,16 +48,12 @@ export const PromotedLBPPoolIds: {
...
-	/*
-		{
-			poolId: '21',
-			name: 'Regen Network',
-			baseDenom: DenomHelper.ibcDenom([{ portId: 'transfer', channelId: 'channel-8' }], 'uregen'),
-			destDenom: DenomHelper.ibcDenom([{ portId: 'transfer', channelId: 'channel-0' }], 'uatom'),
-		},
-		 */
+	{
+		poolId: '600',
```
Comment (Thunnini) [inline src/config.ts:53]: Please check that the pool is correct.

File: src/config.ts [hunk 3]
```diff
@@ -458,6 +454,12 @@ export const IBCAssetInfos: {
 		destChannelId: 'channel-2',
 		coinMinimalDenom: 'boot',
 	},
+	{
```
Comment (Thunnini) [inline src/config.ts:457]: Please check that the channel info is correct.

File: src/stores/root.ts (hunk 1 above)
Comment (sunnya97) [reply src/stores/root.ts:232]: Is correct.

File: src/config.ts (hunk 2 above)
Comment (sunnya97) [reply src/config.ts:53]: Is correct.

File: src/config.ts (hunk 3 above)
Comment (sunnya97) [reply src/config.ts:457]: Is correct.

**Pull request merged**